src/profile/hooks/useProfileMutations.ts

```diff
@@ -0,0 +1,145 @@
+import {
+  useMutation,
+  FetchResult,
+  Reference,
+  ApolloCache,
+  ApolloError,
+} from '@apollo/client';
+import { loader } from 'graphql.macro';
+import { useContext } from 'react';
+import { Modifiers } from '@apollo/client/cache/core/types/common';
+
+import { UpdateMyProfileVariables } from '../../graphql/generatedTypes';
+import {
+  ProfileRoot,
+  UpdateProfileRoot,
+  UpdateProfileData,
+  AnyObject,
+} from '../../graphql/typings';
+import {
+  FormValues,
+  EditDataType,
+  basicDataType,
+  additionalInformationType,
+} from '../helpers/editData';
+import { updateMutationVariables } from '../helpers/updateMutationVariables';
+import { ProfileContext, ProfileContextData } from '../context/ProfileContext';
+
+const UPDATE_PROFILE = loader('../../profile/graphql/UpdateMyProfile.graphql');
```

Comment:
Is there a reason why these follow different naming conventions than regular const variables?

Reply:
Only reason is that the old Profile UI had them like that the code is moved here.